### ISSUE TITLE
Driver token map computation failure

### ISSUE THREAD

**Issue by kathirsvn** [stargate/jsonapi#661]
The CQL Java driver prints below error once in a while

```java
stargate-jsonapi-d4f85888f-gv25c jsonapi WARN  [s58-admin-0] 2023-11-13 16:58:29,815 NetworkTopologyReplicationStrategy.java:132 - [s58] Error while computing token map for replication settings {class=org.apache.cassandra.locator.NetworkTopologyStrategy, europe-west4=3}: could not achieve replication factor 3 for datacenter europe-west4 (found only 0 replicas).
```

**Comment by jeffreyscarpenter** [comment]
@kathirsvn can we close this now?

**Comment by jeffreyscarpenter** [comment]
Message is no longer occurring, appears to have been fixed by #662
